client/state/themes/selectors/is-theme-allowed-on-site.ts

```diff
@@ -0,0 +1,12 @@
+import siteHasFeature from 'calypso/state/selectors/site-has-feature';
+import { getThemeTierForTheme } from 'calypso/state/themes/selectors/get-theme-tier-for-theme';
+import { IAppState } from 'calypso/state/types';
+
+export function isThemeAllowedOnSite( state: IAppState, siteId: number | null, themeId: string ) {
```

Comment:
Nitpick: Selectors are normally exported in [the index.js file](https://github.com/Automattic/wp-calypso/blob/trunk/client/state/themes/selectors/index.js) too so you can do

```
import { isThemeAllowedOnSite } from 'calypso/state/themes/selectors';
```

instead of

```
import { isThemeAllowedOnSite } from 'calypso/state/themes/selectors/is-theme-allowed-on-site';
```

Reply:
Updated.